src/main/kotlin/notary/NotaryInitialization.kt

```diff
@@ -142,16 +143,13 @@ class NotaryInitialization {
         refundServerEndpoint = RefundServerEndpoint(
             ServerInitializationBundle(CONFIG[ConfigKeys.refundPort], CONFIG[ConfigKeys.ethEndpoint]),
             mock {
-                val request = any<EthRefundContract>()
+                val request = any<EthRefundRequest>()
                 on {
                     performRefund(request)
                 } doReturn EthNotaryResponse.Successful(
                     "signature",
-                    "pub_key"
+                    EthRefund("address", "coin", 66.6)
```

Comment:
Currently amount is BigInteger, so `BigInteger.valueOf(666)`

Reply:
The issue has been fixed already.